include/deal.II/python/cell_accessor_wrapper.h

```diff
@@ -0,0 +1,134 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__cell_accessor_wrapper_h
+#define dealii__cell_accessor_wrapper_h
+
+#include <deal.II/grid/tria_accessor.h>
+
+namespace PyDealII
```

Comment:
We currently have an all lowercase convention on namespaces used in deal.II.
What about simply
```
DEAL_II_NAMESPACE_OPEN
namespace python
{
// ...
}
DEAL_II_NAMESPACE_CLOSE
```

This would be unique and fit into the current scheme.